repose-aggregator/commons/utilities/src/main/java/org/openrepose/commons/utils/http/CommonHttpHeader.java

```diff
@@ -41,11 +41,22 @@
     USER_AGENT("User-Agent"),
     VIA("Via"),
     LOCATION("Location"),
+    VARY("Vary"),
 
     //Content specific
     ACCEPT("Accept"),
     CONTENT_TYPE("Content-Type"),
-    CONTENT_LENGTH("Content-Length");
+    CONTENT_LENGTH("Content-Length"),
+
+    //CORS specific
+    ORIGIN("Origin"),
+    ACCESS_CONTROL_REQUEST_METHOD("Access-Control-Request-Method"),
+    ACCESS_CONTROL_REQUEST_HEADERS("Access-Control-Request-Headers"),
+    ACCESS_CONTROL_ALLOW_ORIGIN("Access-Control-Allow-Origin"),
+    ACCESS_CONTROL_ALLOW_CREDENTIALS("Access-Control-Allow-Credentials"),
+    ACCESS_CONTROL_EXPOSE_HEADERS("Access-Control-Expose-Headers"),
+    ACCESS_CONTROL_ALLOW_METHODS("Access-Control-Allow-Methods"),
+    ACCESS_CONTROL_ALLOW_HEADERS("Access-Control-Allow-Headers");
```

Comment:
D'oh.  I was concerned about putting these in a "common" class, and I thought I looked around to see where else I could put them.  I'll move them to a CorsHttpHeader class.